### ISSUE TITLE
Property wrappers error

### ISSUE THREAD

**Issue by Vinestro89** [CoreOffice/XMLCoder#259]
Following the example given in the documentation:

```
struct Book: Codable {
    @Element var id: Int
}
```

I tried to create an instance of a Book:

`let book = Book(id: 42)`

But I get the following error:

`Cannot convert value of type 'Int' to expected argument type 'Element<Int>'`

To make it work I have to do:

`let book = Book(id: Element(42))`

Is there something I'm doing wrong ? Thank you 😊

**Comment by Joannis** [comment]
No, his is correct as it stands. But I'll keep the issue open as I think we can fix it in the signature.
